Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Company.FunctionApp.csproj

```diff
@@ -13,12 +13,12 @@
   <!--#if (NetCore)-->
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
   <!--#endif -->
-  <!--#if (Framework == "net9.0") -->
-    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="2.0.0-preview1" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="2.0.0-preview1" />
-  <!--#else-->
+  <!--#if (FrameworkShouldUseV1Sdk) -->
     <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.23.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.4" />
+  <!--#else-->
+    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="2.0.0-preview1" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="2.0.0-preview2" />
   <!--#endif -->
     <PackageReference Include="Microsoft.ApplicationInsights.WorkerService" Version="2.22.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.ApplicationInsights" Version="1.2.0" />
```

Comment:
Should we move this into the if-else brace so that we can load the v1 and v2 i.e.

If FrameworkShouldUseV1Sdk:

`<PackageReference Include="Microsoft.Azure.Functions.Worker.ApplicationInsights" Version="1.4.0" />`

> and update 1.2.0 to 1.4.0

Else:

`<PackageReference Include="Microsoft.Azure.Functions.Worker.ApplicationInsights" Version="2.0.0" />`

Same for the fsharp project